man/systemd.special.xml

```diff
@@ -512,7 +512,8 @@
           shall add <varname>Conflicts=</varname> dependencies to this
           unit for their service unit, which is implicitly done when
           <varname>DefaultDependencies=yes</varname> is set (the
-          default).</para>
+          default) along with <varname>Before=</varname> dependency.
+          </para>
```

Comment:
So basically reword it to follow systemd implicit behavior. Ok !

BTW, I found out that .target units also get implicit dependency Conflicts=shutdown.target, but without Before=. But it doesnt seem to be mentioned anywhere (at least not in systemd.target, nor in systemd.special). Should I add mention about this as well ?